src/components/MapView/Analyser/index.tsx

```diff
@@ -128,29 +111,27 @@ function Analyser({ classes }: AnalyserProps) {
     }
   };
 
-  const adminBoundariesExtent = useMemo(() => {
-    if (!boundaryLayerData) {
-      // not loaded yet. Should be loaded in MapView
-      return null;
-    }
-    return bbox(boundaryLayerData.data) as Extent; // we get extents of admin boundaries to give to the api.
-  }, [boundaryLayerData]);
-
-  const statisticOptions = Object.entries(AggregationOperations).map(stat => (
-    <FormControlLabel
-      key={stat[0]}
-      value={stat[1]}
-      control={
-        <Radio className={classes.radioOptions} color="default" size="small" />
-      }
-      label={stat[0]}
-    />
-  ));
+  const statisticOptions = Object.entries(AggregationOperations)
+    .filter(stat => stat[1] !== 'sum') // sum is used only for exposure analysis.
```

Comment:
```suggestion
    .filter(([key,value]) => value !== 'sum') // sum is used only for exposure analysis.
```
Prefer this since its easier to make sense of

Reply:
Also, shouldn't we do equality check with AggregationOperations.Sum?
`value !== AggregationOperations.Sum`

Reply:
Included in f19a896404fe2613fbe1aa59f13aaf1b207b1fab